deps/slfritchie_eflame-a08518142126f5fc541a3a3c4a04c27f24448bae/src/eflame.app.src

```diff
@@ -0,0 +1,13 @@
+{application, eflame,
+ [
+  {description, ""},
+  {vsn, git},
+  {registered, []},
+  {modules, []},
+  {applications, [
+                  kernel,
+                  stdlib
+                 ]},
+  {mod, { eflame_app, []}},
```

Comment:
Please be consistent. If you mean for eflame to be a supervised OTP app then turn it into a `gen_`something. It is misleading to have "pretend" `_app` and `_sup` files.